locality_sensitive_hashing/lsh.py

```diff
@@ -164,14 +166,81 @@ def k_shingles() -> KShingles:
         for i in range(len(doc) - k + 1):
             shingles.add(doc[i : i + k])
 
-        docs_k_shingles.append(shingles)
+        docs_k_shingles.append(np.array(list(shingles)))
     return docs_k_shingles
 
 
+def miller_rabin_test(n: int, k: int = 100) -> bool:
+    """
+    Miller-Rabin test
+    Derived from pseudocode in [Miller-Rabin test](https://en.wikipedia.org/wiki/Miller-Rabin_primality_test).
+
+    Arguments:
+        n: int
+            An odd integer > 2 to be tested for primality
+        k: int
+            The number of rounds of testing to perform
+        seed: int
+            The seed for the random number generator
```

Comment:
Outdated docs? Can't see a `seed` argument.

Reply:
Yep! Moved to the correct place in 755a3f2